README.md

```diff
@@ -74,6 +75,26 @@ Additionally installing `Authorino operator` & `Service Mesh operator` enhances
 
   3. Create [DataScienceCluster](#example-datasciencecluster) CR to enable components
 
+
+### Configuration
+
+- in ODH 2.23.1, we introduced a new feature which allows user to use their own application namespace than default one "opendatahub".
+
+1. for new cluster, as this cluster has not been used for ODH or RHOAI.
+   Here we use namespace A for example as targeted application namespace, please follow below steps before install ODH operator:
+
+   - create namespace A
+   - add label `opendatahub.io/application-namespace: true`  onto namespace A. Only one namespace in the cluster can have this label.
```

Comment:
- add the label `opendatahub.io/application-namespace: true` onto namespace A. Only one namespace in the cluster can have this label.

Reply:
feel free to use suggestion for code review next time. 
i can get that change committed if it is needed. now seems a bit too late :D